source/components/ui/Checklist/index.js

```diff
@@ -69,9 +71,11 @@ class CheckItemComponent extends Component<CheckItemProps, CheckItemState> {
 				</div>
 				{this.state.displayExplanations && this.props.explanations && (
 					<Animate.appear>
-						<div className="ui__ checklist-explanation">
-							{this.props.explanations}
-						</div>
+						<div
+							className="ui__ checklist-explanation"
+							dangerouslySetInnerHTML={{
+								__html: marked(this.props.explanations)
```

Comment:
(Du coup avec ma PR ça deviendra `<Markdown source={this.props.explanations} />`) 😊